### PR TITLE
Feat/ld label html content

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

ld-label now also works with html content:

```html
<ld-label position="right">
  <span>I love to <code>code</code>.</span>
  <ld-checkbox></ld-checkbox>
</ld-label>
```

This is quite handy as developers can now add links and other interactive stuff to labels.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Added visual regression tests (e2e) to all label variants

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-label/test/ld-label.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+              />
+            </svg>
+            <div class="ld-checkbox__box"></div>
+          </div>
+        </label>`)
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot({ allowableMismatchedRatio })
+    })
+  })
+})
```
Comment (renet) [inline src/liquid/components/ld-label/test/ld-label.e2e.ts:161]: As the selectors are not that easy to understand and cases can be overseen, I'd recommend adding tests that test the selectors that explicitly handle (or exclude) ld-input-message components.

File: src/liquid/components/ld-label/readme.md [hunk 2]
```diff
@@ -78,21 +78,56 @@ The default size is small. You can use a slightly bigger label (size medium) by
...
     <input placeholder="jane.doe@example.com" type="email">
   </div>
 </label>
 {% endexample %}
 
+### HTML Content
+
+{% example %}
+<ld-label position="right">
+  <span>I love to <code style="line-height: 0">code</code>.</span>
```
Comment (renet) [inline src/liquid/components/ld-label/readme.md:100]: I'd expect a developer might also want to add this label without the surrounding `<span>`-Element. Also what, if I want to add more than one element before (and/or after) the input element in question?

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-label/ld-label.css [hunk 3]
```diff
@@ -54,8 +60,14 @@ ld-label {
     'input label'
     'message message';
 
-  > :first-child:not(ld-input-message):not(.ld-input-message) {
+  > :first-child:not(ld-input-message):not(.ld-input-message):not(:first-child:not(:last-child)) {
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-label/ld-label.css]: Refactored. There is no more last child first child css code.

File: src/liquid/components/ld-label/ld-label.css [hunk 4]
```diff
@@ -29,7 +29,7 @@ ld-label {
     margin-top: var(--ld-sp-4);
   }
 
-  > :not(ld-input-message):not(.ld-input-message) {
+  > :not(ld-input-message):not(.ld-input-message):not(:first-child:not(:last-child)) {
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-label/ld-label.css]: Refactored. There is no more last child first child css code.

File: src/liquid/components/ld-label/test/ld-label.e2e.ts (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/components/ld-label/test/ld-label.e2e.ts:322]: Done.

File: src/liquid/components/ld-label/readme.md (hunk 2 above)
Comment (borisdiakur) [reply src/liquid/components/ld-label/readme.md:236]: The wrapping element is necessary. I have documented this though in my last commits. It should be fine as this is an edge case scenario anyway.

Comment (borisdiakur) [reply src/liquid/components/ld-label/readme.md:236]: Regarding more than one element before and or after: I currently can not think of any use case. In any case this seems to be even more an edge case which I would ignore for now.

File: src/liquid/components/ld-label/ld-label.css [hunk 5]
```diff
@@ -3,12 +3,13 @@ ld-label {
 }
 
 .ld-label {
+  --ld-label-col-gap: 0.625rem;
```
Comment (renet) [inline src/liquid/components/ld-label/ld-label.css:6]:
```suggestion
  --ld-label-col-gap: var(--ld-sp-6);
```

**Review by renet — APPROVED** [review]

**Pull request merged**